### DISCUSSION TITLE
[autocomplete] Add props "max-length" to component el-autocomplete

### DISCUSSION THREAD

**Discussion by Anastasia226** [element-plus/element-plus#10526] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `autocomplete`

## Description

Component el-autocomplete

### Additional comments

_(empty)_

**Comment by ryuhangyeong** [comment]
hello. A lot of people will feel the need if you make it clear what use case or why those props should be added!

**Reply by longtranhd** [reply]
In autocomplete, we can enter many characters we want. But when submitting, it will occur issue in BE. So, we wanna handle in FE like maxlength input
